test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -327,18 +331,20 @@ public void IntegrationTest(
     {
         Guid requestId = Guid.NewGuid();
 
-        ZipkinExporter exporter = new ZipkinExporter(
-            new ZipkinExporterOptions
+#pragma warning disable CA2000 // Dispose objects before losing scope
+        ZipkinExporter exporter = new(
+            new()
             {
-                Endpoint = new Uri($"http://{this.testServerHost}:{this.testServerPort}/api/v2/spans?requestId={requestId}"),
+                Endpoint = new($"http://{this.testServerHost}:{this.testServerPort}/api/v2/spans?requestId={requestId}"),
                 UseShortTraceIds = useShortTraceIds,
             });
+#pragma warning restore CA2000 // Dispose objects before losing scope
```

Comment:
same question as above, can we use a `using` here?

Reply:
Will be handled by `using var processor = new SimpleActivityExportProcessor(exporter);`